silkworm/sentry/rlpx/server.cpp

```diff
@@ -59,7 +59,11 @@ Task<void> Server::run(
     acceptor.set_option(detail::socket_option::boolean<SOL_SOCKET, SO_REUSEPORT>(true));
 #endif
 
-    acceptor.bind(endpoint);
+    boost::system::error_code ec;
+    acceptor.bind(endpoint, ec);
+    if(ec){
+        SILK_ERROR << "Sentry RLPx server has failed to start. TCP port: "<< port_ << " is busy. Try another port with --port=MMM.";
```

Comment:
* we use exceptions, please catch this as `const boost::system::system_error&`
* the check should examine the ex.code() and/or message and understand that it is related to the busy port
* throw `std::runtime_error` in the end, no need to log here